src/current/cockroachcloud/managing-access.md

```diff
@@ -85,15 +83,11 @@ If you are sure you want to delete the organization, proceed with the following
 
 ## Manage service accounts
 
-{{site.data.alerts.callout_info}}
-The access management model for service accounts is unified with users if the updated model is enabled for your organization. Service accounts created prior to that may still have the following legacy roles:
+The access management model for service accounts is unified with that for users, meaning service accounts may have all of the same [access roles](authorization.html#organization-user-roles), although it should be noted that service accounts and users still differ in the actions they can performed, as only users can access the console, and only service accounts can access the API, and the console and API differ in functionality.
```

Comment:
What does "unified" mean here? Like mirrored? Not sure on that, but suggest breaking this sentence up a little anyway (have kept your unified language for now)
```suggestion
The access management model for service accounts is unified with the [user model](#manage-an-organizations-users). This means that service accounts may have all of the same [access roles](authorization.html#organization-user-roles). However, service accounts and users still differ in the actions they can perform: only users can access the console, and only service accounts can access the API. The console and API differ in functionality.
```